tests/visual/components/product/product.scss

```diff
@@ -0,0 +1,2 @@
+
+@import 'dist/components/product/product';
```

Comment:
Oh nevermind, scratch this. I forgot that the visual test SCSS is intended to be compiled to CSS.

`product.scss`  is in fact the correct filename.